test/debugger/helper-debugger-repl.js

```diff
@@ -34,11 +34,11 @@ let quit;
 
 function startDebugger(scriptToDebug) {
   scriptToDebug = process.env.NODE_DEBUGGER_TEST_SCRIPT ||
-                  common.fixturesDir + '/' + scriptToDebug;
+                  `${common.fixturesDir}/${scriptToDebug}`;
```

Comment:
Like James and Benjamin, I’m fine with leaving this as-is. Leaving `path.join()` refactors for a later PR sounds good to me. (If there’s consensus they should happen.)